client/src/components/infoDrawer/infoFormat.js

```diff
@@ -0,0 +1,194 @@
+  return (
+    <>
+      <H3 className={skeleton ? Classes.SKELETON : null}>Affiliations</H3>
+      <UL>
+        {affiliations.map((item, index) => (
+          <div
+            id={`#afil${index}`}
+            key={item}
+            className={skeleton ? Classes.SKELETON : null}
+          >
+            <sup>{index + 1}</sup>
+            {"  "}
+            {item}
+          </div>
+        ))}
+      </UL>
+    </>
+  );
+};
+
+const renderDOILink = (type, doi, skeleton) => {
+  return (
+    doi && (
+      <>
+        <H3 className={skeleton ? Classes.SKELETON : null}>{type}</H3>
+        <p className={skeleton ? Classes.SKELETON : null}>
+          <a href={doi} target="_blank" rel="noopener">
+            {doi}
+          </a>
+        </p>
+      </>
+    )
+  );
+};
+
+const renderOrganism = (organism, skeleton) => {
+  if (organism)
+    return (
+      <>
+        <H3 className={skeleton ? Classes.SKELETON : null}>Organism</H3>
+        <p className={skeleton ? Classes.SKELETON : null}>{organism}</p>
+      </>
+    );
+  return null;
+};
+
+const NUM_CATEGORIES = 8;
+
+const singleValueCategoriesPlaceholder = Array.from(Array(NUM_CATEGORIES)).map(
+  (_, index) => {
+    return [index, index];
+  }
+);
+
+const renderSingleValueCategories = (singleValueCategories, skeleton) => {
+  if (singleValueCategories.size > 0)
+    return (
+      <>
+        <H3 className={skeleton ? Classes.SKELETON : null}>Dataset Metadata</H3>
+        <UL>
+          {Array.from(singleValueCategories).map((pair) => {
+            if (!pair[1] || pair[1] === "") return null;
+            return (
+              <li
+                className={skeleton ? Classes.SKELETON : null}
+                key={pair[0]}
+              >{`${pair[0]}: ${pair[1]}`}</li>
+            );
+          })}
+        </UL>
+      </>
+    );
+  return null;
+};
+
+const renderLinks = (projectLinks, aboutURL, skeleton) => {
+  if (projectLinks)
+    return (
+      <>
+        <H3 className={skeleton ? Classes.SKELETON : null}>Project Links</H3>
+        <UL>
+          {projectLinks.map((link) => {
+            if (link.link_type === "SUMMARY") return null;
+            return (
+              <li
+                key={link.link_name}
+                className={skeleton ? Classes.SKELETON : null}
+              >
+                <a href={link.link_url} target="_blank" rel="noopener">
+                  {link.link_name}
+                </a>
+              </li>
+            );
+          })}
+        </UL>
+      </>
+    );
+  if (aboutURL)
+    return (
+      <>
+        <H3 className={skeleton ? Classes.SKELETON : null}>More Info</H3>
+        <p>
+          <a
+            className={skeleton ? Classes.SKELETON : null}
+            href={aboutURL}
+            target="_blank"
+            rel="noopener"
+          >
+            {aboutURL}
+          </a>
+        </p>
+      </>
+    );
+  return null;
+};
+
+const InfoFormat = ({
```

Comment:
let's move `singleValueCategoriesPlaceholder` closer to callsite for readability

```ts
const NUM_CATEGORIES = 8;

const singleValueCategoriesPlaceholder = Array.from(Array(NUM_CATEGORIES)).map(
  (_, index) => {
    return [index, index];
  }
);

const InfoFormat = ({
  datasetTitle,
  singleValueCategories = new Map(singleValueCategoriesPlaceholder),
  aboutURL = "thisisabouthtelengthofaurl",
  dataPortalProps = singleValueCategories,
  skeleton = false,
}) => { ... }
```